### PR TITLE
Add Quarto icon

### LINKED ISSUE #9152: Request: Quarto
### Brand Name

Quarto

### Website

https://quarto.org/

### Popularity Metric

Similarweb rank: 275,826
https://www.similarweb.com/website/quarto.org/#overview

### Official Resources for Icon and Color

None

### Additional Comments

1. One variant of the icon
2. GNU GPL v2. 
https://quarto.org/trademark.html

We will grant permission to use the Quarto name and logo for projects that meet the following criteria:
- The primary purpose of your project is to promote the spread and improvement of the Quarto software.
- Your project is non-commercial in nature (it can make money to cover its costs or contribute to non-profit entities, but it cannot be run as a for-profit project or business).
- Your project neither promotes nor is associated with entities that currently fail to comply with the GPL license under which Quarto is distributed.

3. Great resource for building blogs and sharing reproducible analysis.

### Contributing

No

### PR DESCRIPTION
**Pull request by cwickham**
quarto

**Issue:** closes #9152 

**Popularity metric:** Similarweb global rank  \# 203,081

### Checklist

- [X] I updated the JSON data in `_data/simple-icons.json`
- [X] I optimized the icon with SVGO or SVGOMG
- [X] The SVG `viewbox` is `0 0 24 24`

### Description

I took our internal SVG resized it, removed fill, and simplified. I've used #39729E for the brand color as that is what we document internally (e.g. and use for link color on our website). However, the logo itself is usually #74AADB. Let me know if it would be more appropriate to specify #74AADB as the brand color.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12526,6 +12526,12 @@
             "hex": "4695EB",
             "source": "https://design.jboss.org/quarkus"
         },
+        {
+            "title": "Quarto",
+            "hex": "39729E",
+            "source": "https://quarto.org",
```
Comment (uncenter) [inline _data/simple-icons.json:12532]: Only seeing a PNG at this source... we need an SVG source here.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
`#39729E` works as the brand color, is indeed the primary color on the website besides the logo.

**Comment by uncenter** [comment]
Oops! Sorry... wrong button.

File: _data/simple-icons.json (hunk 1 above)
Comment (cwickham) [reply _data/simple-icons.json:12532]: The SVG I used is only hosted internally as far as I can tell. The only thing close is this icon: https://github.com/quarto-dev/quarto/blob/83a2dda63d7f2de7239654fc671f48e9c3f53b81/apps/vscode/assets/icon/qmd.svg?plain=1 

Would that be better to link to?

Comment (uncenter) [reply _data/simple-icons.json:12532]: If it lines up with the icon in this PR, then yeah that works.

**Review by adamrusted — APPROVED** [review]
Perfect, thanks @cwickham! 🎉

**Pull request merged**